src/ORM/Rule/IsUnique.php

```diff
@@ -30,14 +30,23 @@ class IsUnique
      */
     protected $_fields;
 
+    /**
+     * The unique check options
+     *
+     * @var array
+     */
+    protected $_options;
```

Comment:
```suggestion
    protected $_options = ['allowUniqueNulls' => false];
```

Reply:
It would be good to have a blurb here on what options are supported and what they do.